templates/supplier_app/Supplier/bank-info-arg.html

```diff
@@ -1,8 +1,8 @@
+{% load i18n %}
 
 
-
-    <h2 class="fs-title arg arg-bank">Bank Info</h2>
-    <h3 class="fs-subtitle arg arg-bank">Place holder for instructions about bank info</h3>
+    <h2 class="fs-title arg arg-bank">{% trans "Bank Info" %}</h2>
+    <h3 class="fs-subtitle arg arg-bank">{% trans "Ingrese los datos de su cuenta bancaria" %}</h3>
```

Comment:
Shouldn't be this by default in english?

Reply:
That module only appears if the supplier chooses Argentina as the origin of the Company, so only if the owner of the Argentinian company was an english speaking person would that be required.

Still, will update the base messages to english.